### PR TITLE
Remove EAP references unless EAP is directly used.

### LINKED ISSUE #139: Change EAP references to WildFly
* in tests
* EapMode
* etc

### PR DESCRIPTION
**Pull request by istraka**
Fixes #139

### CODE REVIEW

File: AWS-README.md [hunk 1]
```diff
@@ -109,18 +109,18 @@ If WildFly is running in standalone mode:
...
                 port = "mngmtPort"))
-static OnlineManagementClient staticEapClientSpecified;
+static OnlineManagementClient client;
 ```
 
 All values may contain expressions (`${my.property}`):
 - `nameTag` - mandatory, value of `name` tag of EC2 Instance
 - `region` - optional, if empty, `sunstone.aws.region` from `sunstone.properties` will be used
-- mode - optional, default to EapMode.STANDALONE
+- mode - optional, default to WildFlyMode.STANDALONE
```
Comment (mnovak1) [inline AWS-README.md:123]: What about `ServerMode` ? :-)

File: aws/src/test/java/aws/cloudformation/AwsTestConstants.java [hunk 2]
```diff
@@ -8,7 +8,7 @@ public class AwsTestConstants {
     public static final String instanceName = "${non.existing:eapSunstoneInjectInstance}";
     // must be same as in sunstone.properties
     public static final String region = "${ec2.region:us-east-1}";
-    public static final String eapMngmtUser = "${non.existing:admin}";
-    public static final String eapMngmtPassword = "${non.existing:pass.1234}";
-    public static final String eapMngmtPort = "${non.existing:9990}";
+    public static final String wildflyMngmtUser = "${non.existing:admin}";
+    public static final String wildflyMngmtPassword = "${non.existing:pass.1234}";
+    public static final String wildflyMngmtPort = "${non.existing:9990}";
```
Comment (mnovak1) [inline aws/src/test/java/aws/cloudformation/AwsTestConstants.java:13]: Could it be just `MngmtUser` and so on? I think it should be ok considering that people are used to it from Arquillian.

**Review by mnovak1 — COMMENTED** [review]
2 suggestions, otherwise LGTM.

File: aws/src/test/java/aws/cloudformation/AwsTestConstants.java (hunk 2 above)
Comment (istraka) [reply aws/src/test/java/aws/cloudformation/AwsTestConstants.java:13]: Sure.

File: AWS-README.md (hunk 1 above)
Comment (istraka) [reply AWS-README.md:123]: Hmm, I have never heard of it in context of WildFly. But I have found `OperatingMode`, wdyt?

**Pull request merged**